ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -31,6 +36,22 @@ sub Configure($$@) {
     # Define paths for convenience.
     my $base = "/software/components/dirperm";
 
+    my $filesystems = "/system/filesystems";
+    if ($config->elementExists($filesystems)) {
+        my $fs_tree = $config->getElement($filesystems)->getTree();
+        push(@configured_mounts, map {$_->{mountpoint}}
+                                 sort { $a->{mountpoint} cmp $b->{mountpoint} }
+                                 @{$fs_tree});
+    }
+
+    # Get actual mountpoints available on the server into a hash.
+    my $fh = CAF::FileEditor->new(MTAB_PATH, log => $self);
```

Comment:
if you can't use `FileReader, can you do a `$fh->cancel();` right after the new (and add a TODO to replace it ast some point in future). this is just in case, i also have no clue if you can edit mtab or what would happen.